oval-schemas/oval-definitions-schema.xsd

```diff
@@ -1103,6 +1114,33 @@
             -->
         </xsd:restriction>
     </xsd:simpleType>
+    <xsd:simpleType name="SortEnumeration">
+        <xsd:annotation>
+            <xsd:documentation>The SortEnumeration simple type defines basic sorting operations.  Currently 'none', 'alphabetical' and 'alphanumerical' are defined..</xsd:documentation>
+        </xsd:annotation>
+        <xsd:restriction base="xsd:string">
+            <xsd:enumeration value="none">
+                <xsd:annotation>
+					<xsd:documentation>No sorting is performed, the values will remain in the order they came in as.</xsd:documentation>
+                </xsd:annotation>
+			</xsd:enumeration>	
+            <xsd:enumeration value="alphabetical">
```

Comment:
I'd call this "lexical", which I believe is more standard terminology, and typically stands in contrast with "natural" ordering.

For example, given the strings {"a1, "a2", "a12", "1", "2", "12"}, in ascending order*, lexical sorting yields:

"1", "12", "2", "a1", "a2", "a12"

And natural sorting yields:

"1", "2", "12", "a1", "a12", "a2"

* this reminds me, there is a difference between the sort type and the order (ascending vs. descending)

Reply:
I'm fine with that, will update